plugins/UM3NetworkPrinting/src/UM3OutputDevicePlugin.py

```diff
@@ -215,6 +229,24 @@ def addManualDevice(self, address):
 
         self._checkManualDevice(address)
 
+    def _createMachineFromDiscoveredPrinter(self, key: str) -> None:
```

Comment:
A bit of documentation would be nice here (just mentioning: This is the callback that is used if a user wants to add a machine based on something that was discovered)

Reply:
Docs added in d3ea50616